db-engine/src/main/java/com/splicemachine/db/iapi/sql/conn/LanguageConnectionFactory.java

```diff
@@ -124,7 +125,8 @@ Statement getStatement(SchemaDescriptor compilationSchema, String statementText,
 								boolean skipStats,
 								double defaultSelectivityFactor,
 								String ipAddress,
-                                String defaultSchema)
+                                String defaultSchema,
+                                Properties sessionProperties)
```

Comment:
nit: indentation

Reply:
With the introduction of sessionProperties, we may be able to remove skipStats, defaultSelectivityFactor which are session parameters that got passed in separately, and now they are redundant. I don't think it needs to be done in this JIRA. I'll create a new JIRA to do the code cleanup in relevant modules.

Reply:
I opened DB-8331 for the code cleanup.